docs/articles/sde/how-to-use-dev-image.rst

```diff
@@ -6,7 +6,7 @@ How to use development image
 
 Introduction
 -------------
-The dev image is a superset of its corresponding production image for each
+The dev image is a super set of its corresponding production image for each
```

Comment:
This is an unrelated change to the actual commit and should be either a separate commit or unchanged :)

Reply:
I will revert this change and make a separate commit then. Thanks.

Reply:
Superset is a noun that is written together http://www.yourdictionary.com/superset. Not sure why would you change it.